embeds/ge2017-nugget/_source/main.js

```diff
@@ -1,88 +1,142 @@
-        targetContainer.querySelector('.fc-container__inner').appendChild(newTreatList);
-        containerTreats = targetContainer.querySelector('.treats__container');
+    function isContributor() {
+        return !!getCookie(CONTRIBUTION_COOKIE)
     }
 
-    // inject on top if there are treats, at bottom otherwise
-    var firstTreat = containerTreats.querySelector('li:first-child');
-    if (firstTreat) {
-        containerTreats.insertBefore(newTreat, firstTreat);
-        firstTreat.classList.add('treat-separator');
-    } else {
-        containerTreats.appendChild(newTreat);
+    function shouldReaderSeeGoldenNugget() {
+        return !isPayingMember() && !isContributor()
     }
 
+    try {
+
+        if (shouldReaderSeeGoldenNugget()) {
+
+            // find this thrasher
+            var thrasher = document.querySelector('.ge2017-supporter-nugget');
+
+            // the thrasher's section tag
+            var thrasherContainer = closest(thrasher, 'section');
+
+            if (!thrasherContainer) {
+                return;
+            }
+
+            // section where the treat should go
+            var targetContainer = findTargetContainer(thrasherContainer);
+
+            if (!targetContainer) {
+                return;
+            }
+
+            // get thrasher innerHTML
+            var thrasherWrap = document.createElement('div');
+            thrasherWrap.appendChild(thrasher.cloneNode(true));
+            var thrasherHTML = thrasherWrap.innerHTML;
+
+            // clone thrasher into newTreat
+            var newTreat = document.createElement('li');
+            newTreat.classList.add('treats__list-item');
+            newTreat.innerHTML = thrasherHTML;
+
+            // find target container's treats
+            // or make it if it doesnt' exist
+            var containerTreats = targetContainer.querySelector('.treats__container');
+            if (!containerTreats) {
+                var newTreatList = document.createElement('ul');
+                newTreatList.classList.add('treats__container');
+                targetContainer.querySelector('.fc-container__inner').appendChild(newTreatList);
+                containerTreats = targetContainer.querySelector('.treats__container');
+            }
+
+            // inject on top if there are treats, at bottom otherwise
+            var firstTreat = containerTreats.querySelector('li:first-child');
+            if (firstTreat) {
+                containerTreats.insertBefore(newTreat, firstTreat);
+                firstTreat.classList.add('treat-separator');
+            } else {
+                containerTreats.appendChild(newTreat);
+            }
+
+            if (window.guardian.ophan && window.guardian.ophan.record) {
+                 window.guardian.ophan.record({
+                    component: 'thrasher : manifestos compared',
+                    value: 1
+                });
+            }       
+        }
+    } catch (err) {
+
```

Comment:
Let's do something like `console.log('Golden Nugget: ', err)` here to make it easier to notice and debug issues